.kokoro/presubmit/graalvm-native-17-aiplatform.cfg

```diff
@@ -0,0 +1,43 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/graalvm17"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "graalvm17"
+}
+
+# TODO: remove this after we've migrated all tests and scripts
+env_vars: {
+  key: "GCLOUD_PROJECT"
+  value: "gcloud-devel"
+}
+
+env_vars: {
+  key: "GOOGLE_CLOUD_PROJECT"
+  value: "gcloud-devel"
+}
+
+env_vars: {
+  key: "GOOGLE_APPLICATION_CREDENTIALS"
+  value: "secret_manager/java-it-service-account"
+}
+
+env_vars: {
+  key: "SECRET_MANAGER_KEYS"
+  value: "java-it-service-account, java-bigqueryconnection-samples-secrets"
+}
+
+env_vars: {
+  key: "MAVEN_MODULES"
+  value: "java-aiplatform"
+}
+
+env_vars: {
+  key: "TEST_ALL_MODULES"
+  value: "false"
+}
```

Comment:
I'm thinking to remove this env_var in a separate PR since we shouldn't even allow this flag to be toggled in the presubmit kokoro build. We can manually trigger the nightly build in fusion if we want to test all modules.